src/main/java/kr/sesac/aoao/server/dino/controller/DinoController.java

```diff
@@ -0,0 +1,33 @@
+package kr.sesac.aoao.server.dino.controller;
+
+import java.util.Optional;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+import kr.sesac.aoao.server.dino.controller.dto.GetUserDinoResponse;
+import kr.sesac.aoao.server.dino.service.DinoService;
+
+/**
+ * @since 2024.01.19
+ * @author 김은서
+ */
+
+@RestController
+public class DinoController {
+	private final DinoService dinoService;
+	@Autowired
+	public DinoController(DinoService dinoService){
+		this.dinoService = dinoService;
+	}
```

Comment:
```suggestion
@RestController
@RequiredArgsContructor
public class DinoController {

	private final DinoService dinoService;
```

다음과 같이 선언하면 코드가 더 간결해지고, 해당 컨트롤러에 다른 service 가 추가 되더라도 생성자를 수정하지 않아도 된다는 장점이 있습니다 ㅎㅎ